grails-app/views/wps/_asyncRequest.xml.gsp

```diff
@@ -13,7 +13,7 @@
             <wps:Reference mimeType="application/octet-stream" xlink:href="http://geoserver/wps" method="POST">
                 <wps:Body>
                     <wps:Execute version="1.0.0" service="WPS">
-                        <ows:Identifier>gs:NetcdfOutput</ows:Identifier>
+                        <ows:Identifier>gs:${jobIdentifier}</ows:Identifier>
```

Comment:
I wasn't really sure of what to call it so I want with 'identifier' because that is what was in the XML tag. `jobType` is better, though.